symfony/framework-bundle/3.3/config/packages/framework.yaml

```diff
@@ -13,3 +13,11 @@ framework:
     #fragments: ~
     php_errors:
         log: true
+
+    cache:
+        # app cache uses the filesystem by default. Other options include:
+        # app: cache.adapter.apcu
+
+        # app: cache.adapter.redis
```

Comment:
I'd add a comment above this line saying `To use redis, use the eponym adapter and configure your redis connection:`